### PR TITLE
Plugin-server: Remove person properties business logic from db.ts

### PR DESCRIPTION
**Pull request by macobo**
## Changes

I'm working on https://github.com/PostHog/posthog/issues/7154 and saw quite a bit of business logic in db.ts. This PR extracts the logic into a separate file. Will follow up with groups-specific changes in other PRs. :)

## How did you test this code?

Pure refactor + existing tests.

### CODE REVIEW

File: plugin-server/src/worker/ingestion/properties-updater.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                JSON.stringify(person.properties),
+                JSON.stringify(person.properties_last_updated_at),
+                JSON.stringify(person.properties_last_operation || {}),
+                person.id,
+            ]
+        )
+        person.version = Number(updateResult.rows[0].version)
+    })
+
+    if (db.kafkaProducer && person) {
```
Comment (macobo) [inline plugin-server/src/worker/ingestion/properties-updater.ts:53]: Note: This line is pretty odd. We always update kafka even if person properties are not updated? Smells like a bug.

Comment (tiina303) [reply plugin-server/src/worker/ingestion/properties-updater.ts:53]: That's just there to make the type checker happy we're always guaranteed to have a person at this point as if we didn't get a person we would have thrown at line 25. Open to any suggestions how to make this more easily readable/less confusing.

**Review by tiina303 — APPROVED** [review]
Thank you :)

File: plugin-server/src/worker/ingestion/properties-updater.ts (hunk 1 above)
Comment (tiina303) [reply plugin-server/src/worker/ingestion/properties-updater.ts:53]: right yes that's a bug (I thought line 32 handled it, but obviously it doesn't), will put up a PR

**Pull request merged**